ProcessMaker/Repositories/TokenRepository.php

```diff
@@ -225,6 +227,65 @@ private function validateAndSendActionByEmail(ActivityInterface $activity, Token
         }
     }
 
+    /**
+     * Validates the user's email notification settings and sends an email if enabled.
+     *
+     * @param TokenInterface $token The token containing task information.
+     * @param User $user The user to whom the email notification will be sent.
+     * @return mixed|null Returns the result of the email sending operation or null if not sent.
+     */
+    private function validateEmailUserNotification(TokenInterface $token, User $user)
+    {
+        try {
+            Log::Info('User isEmailTaskEnable: ' . $user->email_task_notification);
+            // Return if email task notification is not enabled or email is empty
+            if (!$user->email_task_notification || empty($user->email)) {
+                return null;
+            }
+            // Prepare data for the email
+            $data = $this->prepareEmailData($token, $user);
+
+            // Send Email
+            return (new TaskActionByEmail())->sendAbeEmail($data['configEmail'], $user->email, $data['emailData']);
+        } catch (\Exception $e) {
+            // Catch and log the error
+            Log::error('Failed to validate and send email task notification', [
+                'error' => $e->getMessage(),
+            ]);
+        }
+    }
+
+    /**
+     * Prepares the email data and configuration for sending an email notification.
+     *
+     * @param TokenInterface $token The token containing task information.
+     * @param User $user The user for whom the email data is being prepared.
+     * @return array An associative array containing 'emailData' and 'configEmail'.
+     */
+    private function prepareEmailData(TokenInterface $token, User $user)
+    {
+        $taskName = $token->element_name ?? '';
+        // Prepare the email data
+        $emailData = [
```

Comment:
We need add an extra parameter in the Screen Email requires an image that is used in this ticket
[FOUR-20675](https://processmaker.atlassian.net/browse/FOUR-20675), the image is already uploaded (processmaker-login-svg)